### PR TITLE
EscCol: use the K-12 MG1655 strain, rather HUSEC2011 (ensembl default).

### PR DESCRIPTION
**Pull request by grahamgower**
This better matches the parameter values we're using, but at the cost
of increased complexity. Note that we're still using parameter values
that do not necessarily apply to K-12.

Part of the species QC process in #542.

But is using this particular set of parameters for EscCol even useful to anyone? Fun fact: attemptying to simulate this (`stdpopsim EscCol 10 -o /dev/null`) eats all your memory until the process gets killed. It would be nice to have an E. coli expert weigh in here.

### CODE REVIEW

File: stdpopsim/catalog/ensembl_info.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # File autogenerated from Ensembl REST API. Do not edit.
-release = 100
+release = 101
```
Comment (jeromekelleher) [inline stdpopsim/catalog/ensembl_info.py:2]: Can you confirm that the script ran for all genomes? We don't want to have some at release 100 and others at 101 (although I'm sure there's probably no changes).

File: stdpopsim/catalog/EscCol/__init__.py [hunk 2]
```diff
@@ -40,39 +37,52 @@
...
+            _wielgoss_et_al.because(stdpopsim.CiteReason.MUT_RATE),
             ],
         assembly_citations=[
             _blattner_et_al.because(stdpopsim.CiteReason.ASSEMBLY)])
 
-_species = stdpopsim.Species(
+
+# Nasty hack to overwrite ensembl_id property. Monkey patching won't work in
+# this case, as ensembl_id is a property without a setter.
+class Ecoli(stdpopsim.Species):
```
Comment (jeromekelleher) [inline stdpopsim/catalog/EscCol/__init__.py:60]: Should we have a different way of specifying the ensembl_id? It seems like this stuff might be common,  and it's only used for doing the maintenance stuff, so feel free to change it if it's getting in the way.

**Review by jeromekelleher — APPROVED** [review]
LGTM. Might be worth taking a look at how ensembl_id is defined/used - could be easier to change it than the current approach. It's not that big a deal to just drop a literal definition into the species files, e.g.

File: stdpopsim/catalog/ensembl_info.py (hunk 1 above)
Comment (grahamgower) [reply stdpopsim/catalog/ensembl_info.py:2]: > Can you confirm that the script ran for all genomes?

Yep.

File: stdpopsim/catalog/EscCol/__init__.py (hunk 2 above)
Comment (grahamgower) [reply stdpopsim/catalog/EscCol/__init__.py]: I changed ensembl_id to be a class attribute rather than a property.

**Pull request merged**